alerting/src/main/kotlin/com/amazon/opendistroforelasticsearch/alerting/model/MonitorRunResult.kt

```diff
@@ -133,6 +134,7 @@ data class TriggerRunResult(
 ) : Writeable, ToXContent {
 
     @Throws(IOException::class)
+    @Suppress("UNCHECKED_CAST")
```

Comment:
I fixed this warning only for that field rather than for complete method. https://github.com/opendistro-for-elasticsearch/alerting/blob/7320bc6071804f810d4245dda701ebebc9cbeb3c/alerting/src/main/kotlin/com/amazon/opendistroforelasticsearch/alerting/model/MonitorRunResult.kt#L150 You might get merge conflicts.

Reply:
Changed this to use your implementation.